tests/api/test_opencode_provider_service.py

```
@@ -328,6 +382,62 @@ async def post(self, url: str, headers=None, json=None, data=None):  # type: ign
+    class _FakeAsyncClient:
+        def __init__(self, timeout: float) -> None:
+            self.timeout = timeout
+
+        async def __aenter__(self) -> _FakeAsyncClient:
+            return self
+
+        async def __aexit__(self, exc_type, exc, tb) -> bool:  # type: ignore[no-untyped-def]
+            del exc_type, exc, tb
+            return False
+
+        async def post(self, url: str, headers=None, json=None, data=None):  # type: ignore[no-untyped-def]
```

Ruff (0.15.14)

[warning] Missing return type annotation for private function `post`

(ANN202)